### ISSUE TITLE
Queries with StudyInstanceUIDs that don't exist in the Beams table cause crash (e.g. Brachy plans)

### ISSUE THREAD

**Issue by cutright** [cutright/DVH-Analytics#66]
DVHA <=v0.7.6 does not properly catch tx modality (at least for BrachyVision) and leads to a `None` for the tx_modality.  When querying for tx_modality = None, this happens.

The `v0.7.7dev` branch has a fix for the bracy tx_modality issue.  But there may be a larger issue involving queries of None-Type values in the database.

```
(venv) C:\Users\dcutright\PycharmProjects\DVH-Analytics>python dvha_app.py
Traceback (most recent call last):
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\main.py", line 666, in exec_query_button
    self.exec_query()
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\main.py", line 688, in exec_query
    DVH(dvh_condition=dvh_str, uid=uids, dvh_bin_width=self.options.dvh_bin_width)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\models\dvh.py", line 70, in __init__
    self.rx_dose = self.get_plan_values('rx_dose')
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\models\dvh.py", line 134, in get_plan_values
    values[i] = 'None'
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\models\dvh.py", line 134, in <listcomp>
    values[i] = 'None'
ValueError: '<redacted study instance uid>' is not in list

```

**Comment by cutright** [comment]
Right now, this only happens with a set of BrachyVision plans I just picked up, some of them have a rx_dose of zero, something else may be going on.

When `dvha.models.dvh.DVH` is created, the provided `uid` list *should* have already been validated with `dvha.main.DVHAMainFrame.get_query`.  But somehow `get_query` is passing StudyInstanceUIDs that are not found in the query return.

After adding code for inserting default values when a uid is not found (see previous commit referenced in this thread), I'm now getting an error in `dvha.tools.stats.__map_data`.

**Comment by bastula** [comment]
From my experience BV plans never come with RX dose. This was valid until BV 11. Not sure about later versions.

**Comment by cutright** [comment]
DVHA has an assumption (somewhere) that all SQL tables contain at least one row for every StudyInstanceUID.  If I insert a dummy beam row for these brachy plans, DVHA no longer crashes.

Now the Correlation tab fails due to some `NaN` values, but at least it's not a crash.

**Comment by cutright** [comment]
Fixed in v0.7.7, will be released today.
